resources/assets/js/app.js

```diff
@@ -134,30 +133,34 @@ $(document).ready(function(){
       data: data,
       processData: false,
       contentType: false,
-      beforeSend: function(){
-        $("body").css("cursor", "progress");
-        $("div").removeClass("has-error");
-        $(".help-block").remove();
+      beforeSend: function () {
+        $('body').css('cursor', 'progress');
+        $('.has-error').removeClass('has-error');
+        $('.help-block').remove();
       },
-      success: function(d){
-        $("body").css("cursor", "auto");
-
-        $.each(d.errors, function(key, row){
-                                        //Scroll to first error
-                                        if (Object.keys(d.errors).indexOf(key) === 0) {
-                                            $('html, body').animate({
-                                                scrollTop: $("[data-name='"+key+"']").parent().offset().top
-                                                        - $('nav.navbar').height() + 'px'
-                                            }, 'fast');
-                                        }
-
-          $("[data-name='"+key+"']").parent().addClass("has-error");
-          $("[data-name='"+key+"']").parent().append("<span class='help-block' style='color:#f96868'>"+row+"</span>")
+      success: function (d) {
+        $('body').css('cursor', 'auto');
+        $.each(d.errors, function (inputName, errorMessage) {
+          // Using ^ 'starts with' operator to also look for input arrays
+          var $inputElement = $("[name^='" + inputName + "']"),
```

Comment:
My only hesitation with the 'starts with' operator is the case of `address` vs `address2`.

Reply:
Yes, In this case it will return a collection of objects, and then get the first. But just realized that the first element of the collection may not be the same object I'm looking for... Will try to come up with some better solution for this.